metricbeat/module/postgresql/_meta/kibana/7/dashboard/Metricbeat-postgresql-overview.json

```diff
@@ -0,0 +1,1324 @@
+        },
+        {
+          "id": "e2b28ce0-b908-11e9-a579-f5c0a5d81340",
+          "name": "panel_9",
+          "type": "visualization"
+        }
+      ],
+      "type": "dashboard",
+      "updated_at": "2019-08-07T11:48:08.889Z",
+      "version": "WzgzMywxXQ=="
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [],
+            "query": {
+              "language": "kuery",
+              "query": ""
+            }
+          }
+        },
+        "title": "[Metricbeat PostgreSQL] Query Latency",
+        "uiStateJSON": {},
+        "version": 1,
+        "visState": {
+          "aggs": [],
+          "params": {
+            "axis_formatter": "number",
+            "axis_position": "left",
+            "axis_scale": "normal",
+            "default_index_pattern": "metricbeat-*",
+            "default_timefield": "@timestamp",
+            "filter": {
+              "language": "kuery",
+              "query": ""
+            },
+            "id": "919c5570-b796-11e9-8ed3-ef1959e6b366",
+            "index_pattern": "metricbeat-*",
+            "interval": "auto",
+            "series": [
+              {
+                "axis_position": "right",
+                "chart_type": "line",
+                "color": "rgba(0,156,224,1)",
+                "fill": 0,
+                "formatter": "number",
+                "id": "919c5571-b796-11e9-8ed3-ef1959e6b366",
+                "label": "Query Latency",
+                "line_width": 2,
+                "metrics": [
+                  {
+                    "field": "postgresql.statement.query.time.total.ms",
+                    "id": "919c7c80-b796-11e9-8ed3-ef1959e6b366",
+                    "type": "max"
+                  },
+                  {
+                    "field": "919c7c80-b796-11e9-8ed3-ef1959e6b366",
+                    "id": "9e553c60-b79f-11e9-9029-a9d302b79ec2",
+                    "type": "derivative",
+                    "unit": "1s"
+                  },
+                  {
+                    "field": "9e553c60-b79f-11e9-9029-a9d302b79ec2",
+                    "id": "a58b6590-b79f-11e9-9029-a9d302b79ec2",
+                    "type": "positive_only",
+                    "unit": ""
+                  }
+                ],
+                "point_size": 0,
+                "separate_axis": 0,
+                "split_mode": "terms",
+                "stacked": "none",
+                "terms_field": "postgresql.statement.query.text",
+                "terms_order_by": "919c7c80-b796-11e9-8ed3-ef1959e6b366",
+                "terms_size": "10",
+                "value_template": "{{value}}"
+              }
+            ],
+            "show_grid": 1,
+            "show_legend": 1,
+            "time_field": "@timestamp",
+            "type": "timeseries"
+          },
+          "title": "[Metricbeat PostgreSQL] Query Latency",
```

Comment:
This needs to be updated to the format(https://github.com/elastic/beats/pull/13168): Query Latency [Metricbeat PostgresSQL] and same with other title 😬

Reply:
updated